package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "npm-register",
   "description": "Your own private npm registry and backup server",
-  "version": "2.5.2",
+  "version": "2.6.0",
```

Comment:
@Ianfeather revert this file, I will make sure it's updated when we republish.

Reply:
Will do